R/bootstrap-layout.R

```diff
@@ -231,8 +231,12 @@ column <- function(width, ..., offset = 0) {
     stop("column width must be between 1 and 12")
```

Comment:
Maybe we should consider a `bs4::column()` that doesn't require `width` to take advantage of the new auto-widths

Reply:
Yes, I started working on that. Also wanted a way to specify different widths for different breakpoints (sm, lg, etc)

Reply:
There are equal width columns and also the auto width (width of content) columns. I referred to these values as `"equal"` and `"content"` in yonder, but I could see changing the values for the sake of continuity across packages. 

As for breakpoints I opted for `c(sm = 1, lg = "auto", default = 12)`, where `default` could also be `xs`.